README.md

```diff
@@ -1,6 +1,6 @@
 # Virtuoso Design System
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![All Contributors](https://img.shields.io/badge/all_contributors-2-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/badge/all_contributors-12-orange.svg?style=flat-square)](#contributors-)
```

Comment:
how about blue instead of orange?

`https://img.shields.io/badge/all_contributors-12-blue.svg?style=flat-square`

![blue contributors](https://img.shields.io/badge/all_contributors-12-blue.svg?style=flat-square)

Reply:
This is what the script did. Let's look at changing this later. ;)